feature/experimental/p4rt/otg_tests/lldp_packetout_test/lldp_packetout_test.go

```diff
@@ -0,0 +1,457 @@
+	leader := args.leader
+
+	// Insert wbb acl entry on the DUT
+	if err := programmTableEntry(ctx, t, leader, args.packetIO, false); err != nil {
+		t.Fatalf("There is error when programming entry")
+	}
+	// Delete wbb acl entry on the device
+	defer programmTableEntry(ctx, t, leader, args.packetIO, true)
+
+	packetOutTests := []struct {
+		desc       string
+		client     *p4rt_client.P4RTClient
+		expectPass bool
+	}{{
+		desc:       "PacketOut from Primary Controller",
+		client:     leader,
+		expectPass: true,
+	}}
+
+	for _, test := range packetOutTests {
+		t.Run(test.desc, func(t *testing.T) {
+			// Check initial packet counters
+			port := sortPorts(args.ate.Ports())[0].ID()
+			counter0 := gnmi.Get(t, args.ate.OTG(), gnmi.OTG().Port(port).Counters().InFrames().State())
+
+			packets := args.packetIO.GetPacketOut(portID, false)
+			sendPackets(t, test.client, packets, packetCount)
+
+			// Wait for ate stats to be populated
+			time.Sleep(60 * time.Second)
+
+			// Check packet counters after packet out
+			counter1 := gnmi.Get(t, args.ate.OTG(), gnmi.OTG().Port(port).Counters().InFrames().State())
+
+			// Verify InPkts stats to check P4RT stream
+			t.Logf("Received %v packets on ATE port %s", counter1-counter0, port)
+
+			if test.expectPass {
+				if counter1-counter0 < uint64(float64(packetCount)*0.95) {
+					t.Fatalf("Not all the packets are received.")
+				}
+			} else {
+				if counter1-counter0 > uint64(float64(packetCount)*0.10) {
+					t.Fatalf("Unexpected packets are received.")
+				}
+			}
+		})
+	}
+}
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// sortPorts sorts the ports by the testbed port ID.
+func sortPorts(ports []*ondatra.Port) []*ondatra.Port {
+	sort.Slice(ports, func(i, j int) bool {
+		idi, idj := ports[i].ID(), ports[j].ID()
+		li, lj := len(idi), len(idj)
+		if li == lj {
+			return idi < idj
+		}
+		return li < lj // "port2" < "port10"
+	})
+	return ports
+}
+
+// configInterfaceDUT configures the interface with the Addrs.
+func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes) *oc.Interface {
+	i.Description = ygot.String(a.Desc)
+	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	if *deviations.InterfaceEnabled {
+		i.Enabled = ygot.Bool(true)
+	}
+
+	s := i.GetOrCreateSubinterface(0)
+	s4 := s.GetOrCreateIpv4()
+	if *deviations.InterfaceEnabled {
+		s4.Enabled = ygot.Bool(true)
+	}
+	s4a := s4.GetOrCreateAddress(a.IPv4)
+	s4a.PrefixLength = ygot.Uint8(ipv4PrefixLen)
+
+	return i
+}
+
+// configureDUT configures port1 and port2 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	d := gnmi.OC()
+
+	p1 := dut.Port(t, "port1")
+	i1 := &oc.Interface{Name: ygot.String(p1.Name())}
+	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), configInterfaceDUT(i1, &dutPort1))
+
+	p2 := dut.Port(t, "port2")
+	i2 := &oc.Interface{Name: ygot.String(p2.Name())}
+	gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), configInterfaceDUT(i2, &dutPort2))
```

Comment:
Needs a couple more deviations after this from https://github.com/openconfig/featureprofiles/blob/86c1754184d754797f3d7c5d5f5a6a048ee33b92/feature/experimental/p4rt/ate_tests/lldp_packetout_test/lldp_packetout_test.go#L232

Reply:
Added the deviation